### ISSUE TITLE
What about non GitHub repos

### ISSUE THREAD

**Issue by maelle** [cynkra/fledge#267]
Should fledge really be only closely coupled to GitHub.

**Comment by krlmlr** [comment]
The default can be GitHub, we could offer a fallback for non-GH repos. Low priority, also how to detect GH Enterprise?

**Comment by maelle** [comment]
Short-term, add a note that GitHub usage is assumed for the release functions.

**Comment by maelle** [comment]
Noting that `is_merge_commit()` only recognizes the pattern for GitHub PRs so that is where the "filtering" happens.

**Comment by maelle** [comment]
probably some getwd() clash

**Comment by maelle** [comment]
Maybe after #686 is merged in its current state, a new PR should focus on this issue: making sure the release functions don't fail for non GitHub repos.

A short-term fix would be to just not create PRs for non GitHub repos. But eventually, should we support the opening and merging of PR equivalents on at least GitLab?

**Comment by maelle** [comment]
First problem I'm noticing: Quarto blogs have a search bar, which is nice, but we'd need to disable it as it'd mistakenly look like search on the whole website.
